scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/authentication/ManageUsers.java

```diff
@@ -527,12 +666,12 @@ public String toString() {
     static class UserInfo {
         private String login;
         private String password;
-        private String[] groups;
+        private Collection<String> groups;
```

Comment:
Initialize with an empty collection to prevent some _null_ checks in other parts of the code?